Tests/scripts/validate_files_structure.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+### this script ensure all files starts/ends with the right prefix/suffix
+
+foundWrongName=false
+
+# ensures all files in dir starting/ending with prefix/suffix
+# if schema file path provided - validate all files in dir match the schema file
+validateFilesStructure() {
+    dir="$1"
+    prefix="$2"
+    suffix="$3"
+    validate_schema_file="$4"
+
+    echo "Starting validate $dir..."
+     # iterate all files in dir
+    for entry in $dir/*
+    do
+        filename=$(basename $entry)
+        if ! [[ $filename == $prefix* ]]
+        then
+        	echo "file $dir/$filename should start with $prefix"
+        	foundWrongName=true
+        fi
+
+        if ! [[ $filename == *$suffix ]]
+        then
+        	echo "file $dir/$filename should end with $suffix"
+        	foundWrongName=true
+        fi
+        if ! [ -z "$validate_schema_file" ]
+        then
+            python Tests/validate_schema.py "$dir/$filename" "Tests/schemas/$validate_schema_file.yml" d
+            if [[ $? -ne 0 ]]
+            then
+                foundMissingField=true
+            fi
+        fi
+    done
+
+    echo "Finished validate $dir"
+}
+
+validateFilesStructure Integrations integration- .yml "" "integration"
```

Comment:
what is the double quatations?
dir: $1 = Integrations
prefix: $2 = integration-
suffix: $3 = .yml
validate_schema_file: $4 = ""

Reply:
same thing in report and script